presto-hive-metastore/src/main/java/com/facebook/presto/hive/BucketFunctionType.java

```diff
@@ -16,5 +16,6 @@
 public enum BucketFunctionType
 {
     HIVE_COMPATIBLE,
-    PRESTO_NATIVE
+    PRESTO_NATIVE,
+    HIVE_CLUSTERING
```

Comment:
maybe call it ZORDER_CLUSTERING given hive doesn't have the notion of clustering